kimchi/src/tests/foreign_field_mul.rs

```diff
@@ -72,6 +69,29 @@ fn pallas_sqrt() -> BigUint {
     pallas_max().sqrt()
 }
 
+// Compute intermediate sums
+fn compute_intermediate_sums<F: PrimeField, T: crate::circuits::expr::constraints::ExprOps<F>>(
```

Comment:
Delete, no longer relevant to the gate

Reply:
Then I will also remove the test that was using this, as it is meant to exploit some witness values that are no longer relevant.